cwl/src/test/scala/wdl4s/cwl/FullyQualifiedNameSpec.scala

```diff
@@ -12,10 +12,10 @@ class FullyQualifiedNameSpec extends FlatSpec with Matchers {
   }
 
   "workflow step output id " should "get filename, id, and step" in {
-    val wfid = WorkflowStepOutputIdReference("file#step/id")
+    val wfid = WorkflowStepIOId("file#step/id")
```

Comment:
Out of interest, why `IOId`?

Reply:
Because it can be used to parse both an input id or an output id

Reply:
Hmm, IMO it's a little confusing since it's nothing to do with async `IO`... but I don't really mind. Don't let me hold up the PR!

Reply:
Hopefully `WorkflowStepInputOrOutputId` is a bit better :)